src/terrain/routes/instantlaunches.clj

```diff
@@ -0,0 +1,26 @@
+(ns terrain.routes.instantlaunches
+  (:use [common-swagger-api.schema]
+        [ring.util.http-response :only [ok]]
+        [terrain.routes.schemas.instantlaunches]
+        [terrain.auth.user-attributes :only [current-user]]
+        [terrain.clients.app-exposer :only [latest-instant-launch-mappings-defaults]]
+        [terrain.util :only [optional-routes]])
+  (:require [terrain.util.config :as config]
+            [clojure.tools.logging :as log]))
+
+(defn instant-launch-routes
+  []
+  (optional-routes
+   [config/app-routes-enabled]
+
+   (context "/instantlaunches" []
+     :tags ["instantlaunches"]
```

Comment:
This could probably use a description in the main swagger `app` config:
https://github.com/cyverse-de/terrain/blob/a9badf59e504c8c697d766c3364957cabfba1165/src/terrain/routes.clj#L250

Reply:
Oops, sorry. Looks like you commented on this right around the same time that I merged it. I'll work on getting a description added in.

Reply:
Yeah, I commented right after the merge. I was slacking and didn't review it sooner 😅